docs/components/field/demo/base-demo.md

```diff
@@ -0,0 +1,19 @@
+```hbs template
+<Form::Field as |field|>
+  <field.Label>First name</field.Label>
```

Comment:
In headless-form, I yielded components using a lowercase name, for no particular reason. Should we try to establish a convention (within CS at least)?

I don't have any strong feelings for one or the other, has anyone of you?

Reply:
I always go with PascalCase for components 🤷

Reply:
But for the `yielded` part...here it is lowercase like `field`:

```
<Form::Field as |field|>
```